python/social_bot/models/pioneer2dx_noplugin/model.sdf

```diff
@@ -62,6 +62,32 @@
         </material>
       </visual>
     </link>
+    <link name="camera">
```

Comment:
It seems that you cannot use "camera" here. Changing back to "link" can work.

Reply:
Yes. Change it to "camera_link".